src/libmongoc/src/mongoc/mongoc-collection.h

```diff
@@ -156,6 +156,23 @@ MONGOC_EXPORT (mongoc_cursor_t *)
 mongoc_collection_find_indexes_with_opts (mongoc_collection_t *collection,
                                           const bson_t *opts)
    BSON_GNUC_WARN_UNUSED_RESULT;
+
+typedef struct _mongoc_index_model_t mongoc_index_model_t;
+
+MONGOC_EXPORT (mongoc_index_model_t *)
+mongoc_index_model_new (const bson_t *keys, const bson_t *opts);
+
+MONGOC_EXPORT (void) mongoc_index_model_destroy (mongoc_index_model_t *model);
+
+MONGOC_EXPORT (bool)
+mongoc_collection_create_indexes_with_opts (mongoc_collection_t *collection,
+                                            mongoc_index_model_t **models,
```

Comment:
```suggestion
                                            mongoc_index_model_t *const *models,
```
Pointers may be `const`, e.g.:
```c
mongoc_index_model_t *const im = mongoc_index_model_new (keys, NULL);
if (mongoc_collection_create_indexes_with_opts (
         coll, &im, 1, NULL /* opts */, NULL /* reply */, &error)) {
   // ...
}
```